packages/dialtone-vue2/components/filter_pill/filter_pill.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <dt-popover
+    class="d-filter-pill"
+    data-qa="dt-filter-pill"
```

Comment:
We probably want to pass through "placement" and "fallbackPlacements" prop since they're very commonly used.

placement can default to "bottom-start"
fallbackPlacements can default to ["top-start", "auto"]

Reply:
A couple other props I think we'll need to pass through

appendTo
maxHeight
maxWidth
padding

Reply:
Makes sense, I wasn't thinking on those, thanks!